src/pages/login/BrowserImport.tsx

```diff
@@ -106,8 +108,8 @@ const BrowserImport: FC<Props> = ({ sendPfx }) => {
               </li>
               <li className="p-list__item">
                 Click the <code>Import</code> button and select the{" "}
-                <code>lxd-ui.pfx</code> file you just downloaded. Confirm an
-                empty password.
+                <code>lxd-ui.pfx</code> file you just downloaded. Enter your
+                password, or leave the field empty if you have not set one.
```

Comment:
I tested on both Chrome and Edge on Linux. The password popup always appears, even if no password was set - as the old copy was also hinting, by the way.
I don't have a Windows machine at hand to test there, but I would assume the behaviour to be the same since they are the same browsers.

Reply:
Windows is special, because there the cert is imported by the OS, so the cert management is not done by the browsers.

Reply:
I think you're mixing up Windows and Mac. Windows should be exactly the same as Linux, as also confirmed by the text in the tutorials for Windows (Chrome and Edge).

Reply:
Nope, that is windows. That is the reason we have the `windowsDialogSteps` reused for edge and chrome on windows.

Reply:
I think we're saying the same thing without understanding each other. Anyway, I'm checking on Windows too now. But if the old copy was "Confirm an empty password", it should be clear already that a password is prompted also when none was set.

Reply:
Just checked on Windows. As hinted by the old copy, the password prompt is always present in the import wizard.

Reply:
Thanks for checking!